scp/canned.php

```diff
@@ -37,6 +37,12 @@
    )),
 ));
 
+// Set fields' attachments so exsting files stay put
```

Comment:
Canned response might not necessarily have attachments.
Canned response might not necessarily have attachments. 

```php
if ($canned 
    && $canned->attachments
    && ($attachments = $canned_form->getField('attachments'))) {
     $attachments->setAttachments($canned->attachments);
 }
```